pandas/io/parsers.py

```diff
@@ -2557,7 +2563,8 @@ def _infer_columns(self):
 
                         while cur_count > 0:
                             counts[col] = cur_count + 1
-                            col = "%s.%d" % (col, cur_count)
+                            col = '{column}.{count}'.format(
+                                column=u''.join(col), count=cur_count)
```

Comment:
I think there is a problem with ASCII encoding by python 2.7 on Windows. I've tried to solve this problem by changing some codes. But it does not work.